controllers/awsmachinetemplate_controller.go

```diff
@@ -253,7 +253,7 @@ func (r *AWSMachineTemplateReconciler) reconcileNormal(ctx context.Context, iamS
 				return ctrl.Result{}, errors.WithStack(err)
 			}
 
-			cloudFrontDomain := key.CloudFrontAlias(baseDomain)
+			cloudFrontDomain := key.IRSADomain(baseDomain, awsCluster.Spec.Region, accountID, clusterName)
```

Comment:
Just a nitpick: the value of this variable might NOT be a `cloudfront domain` so it would be better for readability to rename it. but not blocking OFC